purity/gc/gc.go

```diff
@@ -275,7 +275,7 @@ func (c *collector) markReferencedBlobs(ar *ppb.ActionResult) error {
 	}
 	outputSize, digests, err := c.outputs(result)
 	if err != nil {
-		log.Warning("Couldn't download output tree for %s, continuing anyway: %s", ar.Hash, err)
+		log.Debug("Couldn't download output tree for %s, continuing anyway: %s", ar.Hash, err)
```

Comment:
The reason I have changed this to debug is because it makes the output unreadable in the terminal. It mainly gets ran on a bob though so this doesn't really matter so would be happy to change it back if you think that's better?